examples/correlation_alignment_example.py

```diff
@@ -76,8 +76,6 @@
         logger.info(strm("Rough center is:", rough_center))
         data.setaxis("t2", lambda x: x - rough_center).register_axis({"t2": 0})
         data.ft("t2")
-        mysgn = (select_pathway(data,signal_pathway).C.real.sum("t2").run(np.sign))
-        data *= mysgn
         data.ift("t2")
```

Comment:
I think it's best not to do the sign flip anywhere except in the immediate vicinity of the correlation function -- see [the change I had made here](https://github.com/jmfrancklab/proc_scripts/pull/52/commits/a1c8c3a813429aa2bd89d8e4f6462e028dd1872f#r670782511)

Reply:
fixed